boards/arm/ip_k66f/ip_k66f.dts

```diff
@@ -95,6 +95,7 @@
 
 &enet {
 	status = "okay";
+	local-mac-address = [00 00 12 13 00 10];
```

Comment:
Why do we have this here, wouldn't all devices have this MAC by default now?

Reply:
This was for the LLDP example. In this case `local-mac-address` can be observed when I do test the LLDP example for this PR.
As a test - I use `lldpd` from Linux, but simultaneously I do ping the eth device. In that way I can check if network stack is working properly when with LLDP I test `SOCK_RAW and AF_PACKET`.

Reply:
> This was for the LLDP example.

If this was for one sample, then we should not have it set here in the board dts file. You can add dt overlays in the sample directory to have these values there.